build_tfm.py

```diff
@@ -18,7 +18,7 @@
 """
 
 import os
-from os.path import join, abspath, dirname, isdir, relpath, split
+from os.path import join, abspath, dirname, exists, isdir, relpath, split
```

Comment:
The generally recommended way to import/use functions from `os.path` is to `import os` and call the functions with the qualified namespace e.g `os.path.join` -- this makes it obvious to experienced Python programmers where these functions have come from, and removes ambiguity from code that calls `os.path.join` and `str.join`.